src/pal/src/file/path.cpp

```diff
@@ -56,7 +56,8 @@ GetFullPathNameA(
      OUT LPSTR *lpFilePart)
 {
     DWORD  nReqPathLen, nRet = 0;
-    LPSTR lpUnixPath = NULL;
+    PathCharString UnixPath;
```

Comment:
Could you please change the Unix.... casing to match other locals (start with lowercase)